### PR TITLE
README cleanup and simplification

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

- Change `docker-compose` commands to `docker compose` built into Docker
- Change to de-emphasize Docker Compose in favor of Helm/Kubernetes
- General grammar and spelling cleanup
- Move component documentation to `docs/<component>/README.md`
- Move Docker Compose details from README.md to `docs/installation.md`
- Remove references to "Beta" mirror node
- Remove irrelevant TimescaleDB docs
- Remove unused REST API performance images
- Simplified mirror node overview

**Related issue(s)**:

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,62 @@
...
 
-- The signature and record files are then uploaded from the nodes to Amazon S3 and Google File Storage.
+### Mirror Nodes
 
-- This mirror node software downloads signature files from either S3 or Google File Storage.
+- This mirror node software downloads signature files from cloud storage.
 - The signature files are validated to ensure at least 1/3 of the nodes in the address book (stored in a `0.0.102` file)
   have the same signature.
-- For each valid signature file, the corresponding record file is then downloaded from the cloud.
+- For each valid signature file, the corresponding record file is then downloaded from the cloud and its hash verified
```
Comment (xin-hedera) [inline README.md:48]:
```suggestion
- For each valid signature file, the corresponding record file is then downloaded from cloud storage and its hash verified
```

just to be consistent with line 45.

File: README.md [hunk 2]
```diff
@@ -77,170 +66,60 @@ Multiple technologies are utilized in the mirror node. The following topics are
...
 
 - [Git](https://git-scm.com/about)
+- [Go](https://golang.org/)
+- [Docker](https://docs.docker.com/engine/reference/commandline/docker/)
+- [Helm](https://helm.sh)
 - [Maven](https://maven.apache.org/guides/getting-started/index.html)
-- [Docker](https://docs.docker.com/)
-  - [docker-compose commands](https://docs.docker.com/compose/reference/overview/)
-  - [docker commands](https://docs.docker.com/engine/reference/commandline/docker/)
 - [NodeJS](https://nodejs.org/en/about/)
```
Comment (xin-hedera) [inline README.md:73]: nit: make it consistent with what's on the official about page
```suggestion
- [Node.js](https://nodejs.org/en/about/)
```

File: docs/installation.md [hunk 3]
```diff
@@ -201,67 +194,121 @@ docker run -e NETWORK=TESTNET \
...
-- Importer
-- Monitor
-- REST API
-- Rosetta API
-
-Containers use the following persisted volumes:
+Docker Compose scripts are provided and can run all the mirror node components. Containers use the following persistent
+volumes:
 
 - `./db` on your local machine maps to `/var/lib/postgresql/data` in the containers. This contains the files for the
```
Comment (xin-hedera) [inline docs/installation.md:200]:
```suggestion
- `./db` on your local machine maps to `/var/lib/postgresql/data` in the db container. This contains the files for the
```

Or `PostgreSQL container`

File: docs/installation.md [hunk 4]
```diff
@@ -201,67 +194,121 @@ docker run -e NETWORK=TESTNET \
...
+#### Web3 API
+
+The Web3 API container will display logs similar to the below at start:
+
+```shell
+c.h.m.web3.Web3Application Started Web3Application in 27.808 seconds
+c.h.m.w.config.LoggingFilter /10.0.0.43 POST http://localhost:8545/web3/v1 in 8 ms: 200 OK
+```
+
+To manually verify the Rosetta API endpoints follow the [operations](/docs/operations.md#verifying-2) details.
```
Comment (xin-hedera) [inline docs/installation.md:310]:
```suggestion
To manually verify the Web3 API endpoints follow the [operations](/docs/operations.md#verifying-2) details.
```

I don't see there is a section for manually verifiying web3 in the linked `operations` doc

File: docs/installation.md [hunk 5]
```diff
@@ -201,67 +194,121 @@ docker run -e NETWORK=TESTNET \
...
+
+### Stopping
 
-You may now power down the docker image itself.
+Shut down the database container via `docker compose down`.
```
Comment (xin-hedera) [inline docs/installation.md:314]:
```suggestion
Shut down the containers via `docker compose down`.
```

**Review by xin-hedera — CHANGES_REQUESTED** [review]

File: docs/installation.md (hunk 5 above)
Comment (steven-sheehy) [reply docs/installation.md]: Updated

File: docs/installation.md (hunk 4 above)
Comment (steven-sheehy) [reply docs/installation.md]: Updated to point to `/docs/web3/README.md#acceptance-tests`

File: docs/installation.md (hunk 3 above)
Comment (steven-sheehy) [reply docs/installation.md]: Updated

File: README.md (hunk 2 above)
Comment (steven-sheehy) [reply README.md]: Updated

File: README.md (hunk 1 above)
Comment (steven-sheehy) [reply README.md]: Updated

File: README.md [hunk 6]
```diff
@@ -1,73 +1,62 @@
...
+- Record files contain the hash of the previous record file, thus creating an unbreakable validation chain.
+- The file is closed on a regular cadence and a signature file is generated by the node for the record file.
+- The record and signature files from each node are then uploaded to Amazon S3 and Google Cloud Storage.
 
-- The signature and record files are then uploaded from the nodes to Amazon S3 and Google File Storage.
+### Mirror Nodes
 
-- This mirror node software downloads signature files from either S3 or Google File Storage.
+- This mirror node software downloads signature files from cloud storage.
 - The signature files are validated to ensure at least 1/3 of the nodes in the address book (stored in a `0.0.102` file)
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply README.md]: Updated

**Review by xin-hedera — DISMISSED** [review]
LGTM

File: README.md [hunk 7]
```diff
@@ -77,170 +67,60 @@ Multiple technologies are utilized in the mirror node. The following topics are
...
-- [Go](https://golang.org/dl/) - v1.17+
-- Java - [AdoptOpenJDK 11](https://adoptopenjdk.net/)
+- [Docker Desktop 4.3+](https://www.docker.com/products/docker-desktop)
 
-### Running Mirror Node
+### Running
 
-To run the mirror node, execute these 3 commands in your terminal.
+For production use, we recommend using Kubernetes and to deploy using our Helm [chart](charts). Hedera managed mirror
+nodes use Kubernetes and Helm for its deployments and is considered the most production-ready. As an alternative for
```
Comment (ijungmann) [inline README.md:87]:
```suggestion
nodes use Kubernetes and Helm for their deployments, and this process is considered the most production-ready. As an alternative for
```

File: README.md [hunk 8]
```diff
@@ -1,73 +1,63 @@
...
-full Hedera node through creation of periodic files that contain processed information (such as account balances or
-transaction records), and have the full trust of the Hedera mainnet nodes. The beta mirror node software reduces the
-processing burden by receiving pre-constructed files from the mainnet, validating those, populating a database and
-providing REST APIs.
-
-### Advantages
-
-- Lower compute, bandwidth requirement
+- Lower compute and bandwidth requirements
 - It allows users to only save what they care about, and discard what they don’t (lower storage requirement)
```
Comment (ijungmann) [inline README.md:28]:
```suggestion
- Allows users to only save the data that they care about (lower storage requirement)
```

File: docs/operations.md [hunk 9]
```diff
@@ -341,7 +257,7 @@ Verify all ports:
 for port in {6551..6560}; do curl -s "http://127.0.0.1:${port}/api/v1/transactions?limit=1" && echo; done
 ```
 
-To setup live monitoring, see [monitoring](../hedera-mirror-rest/monitoring/README.md) documentation.
+To setup live monitoring, see [monitoring](/hedera-mirror-rest/monitoring/README.md) documentation.
```
Comment (ijungmann) [inline docs/operations.md:260]:
```suggestion
To set up live monitoring, see [monitoring](/hedera-mirror-rest/monitoring/README.md) documentation.
```

**Review by ijungmann — APPROVED** [review]
Looks good, some nits and suggestions if you'd like input, but nothing major.

File: docs/installation.md [hunk 10]
```diff
@@ -1,16 +1,11 @@
...
-```console
-docker-compose pull
-```
+The Mirror Node can be run [locally](#running-locally) or via [Docker](#running-via-docker-compose).
 
 ## Building
 
-Ensure you have AdoptOpenJDK 11 installed, then run the following command from the top level directory. This will
-compile a runnable Mirror Node JAR file in the `target` directory.
+To run locally, it'll first need to be built using Java. Ensure you have OpenJDK 11 installed, then run the following
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply docs/installation.md]: Updated

File: docs/installation.md [hunk 11]
```diff
@@ -20,14 +15,12 @@ compile a runnable Mirror Node JAR file in the `target` directory.
...
-the version of its database schema.
+In addition to OpenJDK 11, you will need to install a [PostgreSQL](https://postgresql.org) database and initialize it.
 
-For both databases, since [Flyway](https://flywaydb.org) will manage the database schema, the only required step is to
-run the database initialization script. Locate the SQL script
-at `hedera-mirror-importer/src/main/resources/db/scripts/init.sh` and edit the file to change the name and password
-variables at the top of the file as appropriate. Then make sure the application [configuration](configuration.md)
+Since [Flyway](https://flywaydb.org) will manage the database schema, the only required step is to run the database
+initialization script. Locate the SQL script at `hedera-mirror-importer/src/main/resources/db/scripts/init.sh` and edit
+the file to change the name and password variables at the top of the file as appropriate. Then make sure the
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply docs/installation.md]: Updated

File: docs/installation.md [hunk 12]
```diff
@@ -201,67 +194,121 @@ docker run -e NETWORK=TESTNET \
...
-      ...
-      timescaledb:
-      #    deploy:
-      #      replicas: 0
-      ...
-```
+Before starting, [configure](configuration.md) the application by updating the [application.yml](/application.yml)
+file with the desired custom values. This file is passed to Docker Compose and allows customized configuration for each
+of the mirror node components. The `application.yml` file contents represent the minimal set of fields required to
+configure requester pays and must all be uncommented and filled in.
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply docs/installation.md]: Updated

File: README.md (hunk 7 above)
Comment (steven-sheehy) [reply README.md]: Updated

File: README.md (hunk 8 above)
Comment (steven-sheehy) [reply README.md]: Updated

File: docs/operations.md (hunk 9 above)
Comment (steven-sheehy) [reply docs/operations.md]: Updated

**Review by ijungmann — APPROVED** [review]

**Pull request merged**